src/web_app/html/index_template.html

```diff
@@ -28,6 +28,11 @@
   <link rel="canonical" href="{{ room_link }}">
 {% endif %}
   <link rel="stylesheet" href="/css/main.css">
+  <!-- callstats dependencies: http://www.callstats.io/api/#integrating-with-your-sdk -->
+  <script src="https://api.callstats.io/static/callstats.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jsSHA/1.5.0/sha.js"></script>
```

Comment:
Does that mean we can just omit the $ function/object and the jquery lib? I guess your API documentation will be updated to reflect this.